.travis.yml

```diff
@@ -95,11 +96,13 @@ jobs:
         - JOB_URL=$TRAVIS_JOB_WEB_UR
         - SAUCE_USERNAME=$SAUCE_USERNAME
         - SAUCE_ACCESS_KEY=$SAUCE_ACCESS_KEY
```

Comment:
can remove these saucelab env variables if we are not using them for unit tests.